src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},\
+ wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+
+      return this.coords;
+    },
+    goForward(forward = 1) {
+      if (forward < 0) {
+        return this;
+      }
+      this.coords.y += forward;
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(forward = 1) {
      if (forward > 0) {
        this.coords.y += forward;
      }

      return this;
    },
```

better code style - you return this in all cases, but increase coords only by condition

check all moves and fix it below